python/arcticdb/version_store/library.py

```diff
@@ -491,10 +494,19 @@ def __init__(
             len(lib_set) in [0, 1],
             f"LazyDataFrameCollection init requires all provided lazy dataframes to be referring to the same library, but received: {[lib for lib in lib_set]}",
         )
+        output_format_set = {
+            lazy_dataframe.read_request.output_format for lazy_dataframe in lazy_dataframes
+            if lazy_dataframe.read_request.output_format is not None
+        }
+        check(
+            len(output_format_set) in [0, 1],
+            f"LazyDataFrameCollection init requires all provided lazy dataframes to have the same output_format, but received: {[output_format for output_format in output_format_set]}",
```

Comment:
Can't we use `output_format_set` instead of `[output_format for output_format in output_format_set]`